go.mod

```diff
@@ -414,7 +414,7 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
 )
 
-replace github.com/aquasecurity/trivy-db => github.com/skahn007gl/trivy-db v0.0.0-20240723121440-a488d7f107fe
+replace github.com/aquasecurity/trivy-db => github.com/skahn007gl/trivy-db v0.0.0-20240905044623-a3e59a1f3feb
```

Comment:
You can point to https://github.com/aquasecurity/trivy-db/commit/42b851ca3d8ce1b1b49cb012d228a7ee738496b6 now.